daft/runners/partitioning.py

```diff
@@ -44,10 +44,16 @@ class TableParseCSVOptions:
     Args:
         delimiter: The delimiter to use when parsing CSVs, defaults to ","
         header_index: Index of the header row, or None if no header
+        buffer_size: Size of the buffer (in bytes) used by the streaming reader.
+        chunk_size: Size of the chunks (in bytes) deserialized in parallel by the streaming reader.
+        max_chunks_in_flight: Maximum number of chunks deserialized concurrently by the streaming reader.
```

Comment:
I dont think we need to expose `max_chunks_in_flight` to the user

Reply:
I was thinking that it would be nice to expose an override to the user in case our default is over-throttling, so the user doesn't have to wait for another release in order to see a performance fix, but we can hide this arg if we consider that to be unlikely given our concurrency/parallelism model.